src/util/validatePeer.ts

```diff
@@ -1,14 +1,22 @@
 const url = require('url')
 const IGNORE_LIST = ['localhost', '0.0.0.0', 'local.codius.org']
+import * as reduct from 'reduct'
+import Config from '../services/Config'
+const config = reduct()(Config)
```

Comment:
This creates a whole new reduct, which can cause some serious issues. shallowValidatePeer should either take an injector or a config object as a parameter instead. Or you could make this into a service and use reduct to instantiate it